src/Reko-decompiler.sln

```diff
@@ -1630,8 +1650,8 @@ Global
 		{9E4B5291-EBA1-4127-AC38-E0F822F7E4F1}.TravisRelease|x64.Build.0 = TravisRelease|Any CPU
 		{9E4B5291-EBA1-4127-AC38-E0F822F7E4F1}.TravisRelease|x86.ActiveCfg = TravisRelease|Any CPU
 		{9E4B5291-EBA1-4127-AC38-E0F822F7E4F1}.TravisRelease|x86.Build.0 = TravisRelease|Any CPU
-		{1DBA55A8-6C67-4D16-8137-912B79715973}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{1DBA55A8-6C67-4D16-8137-912B79715973}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{1DBA55A8-6C67-4D16-8137-912B79715973}.Debug|Any CPU.ActiveCfg = Debug|x86
+		{1DBA55A8-6C67-4D16-8137-912B79715973}.Debug|Any CPU.Build.0 = Debug|x86
```

Comment:
Is it correct?

Reply:
Shall review all these AnyCPU and specific x86 configuration settings

Reply:
I'll take a peek at the SLN file.

Reply:
This is the NativeProxy assembly, whose `Any CPU` target is meaningless. NativeProxy exists solely to pass on whether or not x86 or x64 is the target platform to the CMake build which compiles the native components.  The real fix is to remove the _solution_ platform `Any CPU`, since we can't build Reko "platform neutrally" because of the dependency on native binaries.

I will investigate what happens when we remove "Any CPU" from the set of available solution platforms. Note that individual _project_ platforms for the majority of the C# projects will remain `Any CPU`.